spec/factories/cars.rb

```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
+  factory :car do
+    sequence(:brand) { |n| "Brand#{n}" }
+    sequence(:model) { |n| "Model#{n}" }
+    sequence(:engine) { |n| "Engine#{n}" }
+    sequence(:fuel_type) { |n| "FuelType#{n}" }
+    sequence(:license_plate) { |n| "Plate#{n}" }
+    year { Date.today }
```

Comment:
shouldn't it be in format like `2022`?

Reply:
Yes, missed that. I will add validation for the year format in the model.